example/test_runner.dart

```diff
@@ -20,49 +20,53 @@ main() {
       abbr: 'c',
       defaultsTo: 'none',
       help: 'Specify any compilation step (if needed).',
-      allowed: ['none', 'dart2js', 'dartc'],
+      allowed: [
+        'none',
+        'dart2js',
+        'dartc'
```

Comment:
Is this the new dartfmt behavior?  I'm surprised it doesn't require a trailing , to explode this 3-item list like this.